### ISSUE TITLE
Icon request

### ISSUE THREAD

**Issue by jteuber** [dkanada/frost#347]
Buienradar
HMD global camera
NOS
Nextcloud Notes
Our Home

At least Notes and NOS already have icons but are apparently not set up correctly as they don't show up for me.
I don't have enough time to look into this this week but might do that myself at the end of the month.

For some reason I couldn't find the file Turtl produces, but here is the snippet it generates in the other mode:
```
<!-- Buienradar -->
<item component="ComponentInfo{com.supportware.Buienradar/nl.rtl.buienradar.ui.splash.SplashActivity}" drawable="buienradar"/>
<!-- Camera -->
<item component="ComponentInfo{com.hmdglobal.app.camera/com.hmdglobal.app.camera.CameraActivity}" drawable="camera"/>
<!-- NOS -->
<item component="ComponentInfo{nl.nos.app/nl.nos.app.activity.RedirectActivity}" drawable="nos"/>
<!-- Notes -->
<item component="ComponentInfo{it.niedermann.owncloud.notes/it.niedermann.owncloud.notes.SplashscreenActivity}" drawable="notes"/>
<!-- OurHome -->
<item component="ComponentInfo{com.getfairshare.ourhome/com.getfairshare.ourhome.MainActivity}" drawable="ourhome"/>
 ```

**Comment by Donnnno** [comment]
> At least Notes and NOS already have icons but are apparently not set up correctly as they don't show up for me. 

Already fixed NOS in my pr #334 , Nextcloud Notes & HMD global camera are there now too. 

- [] Buienradar
- [x] HMD global camera
- [x] NOS
- [x] Nextcloud Notes
- [] Our Home

**Comment by Donnnno** [comment]
I've created the last tow icons, they will be added in my next pull request.

![image](https://user-images.githubusercontent.com/31142286/91983633-40385900-ed2c-11ea-8c8c-ebbcf73d55b4.png)
